src/server/index.jsx

```diff
@@ -114,16 +123,7 @@ if (process.env.SIMORGH_APP_ENV !== 'local') {
 const sendDataFile = (res, dataFilePath, next) => {
   res.sendFile(dataFilePath, {}, sendErr => {
     if (sendErr) {
-      logger.error(
-        JSON.stringify(
-          {
-            event: 'local_sendfile_error',
-            message: sendErr,
-          },
-          null,
-          2,
-        ),
-      );
+      logger.error(LOCAL_SENDFILE_ERROR, { error: sendErr });
```

Comment:
I like the refactor with the update to the `logger.error` methods, since the JSON strigification can be hidden away and the API is much cleaner.